### PR TITLE
Feature/eventCard

### PR DESCRIPTION
**Pull request by ntnshpig**

### CODE REVIEW

File: src/FakeUpcomingEvents.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    },
+    {
+        title: "AUTO.RIA Race", 
+        date: "12.12.2021",
+        address: "Kharkiv. Maidan constitution",
+        discipline: "Digital motorsport.",
+        status: "National Seria",
+        series: "National Digital Time Attack Series (NS-CTA) 2021"
+    },
+]
```
Comment (VolodymyrDevico) [inline src/FakeUpcomingEvents.js]: new line in the end of document

Comment (VolodymyrDevico) [reply src/FakeUpcomingEvents.js]: use 2 spaces at project

File: src/components/Carousel/Carousel.tsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+export interface CarouselItem {
+  index: number;
+  jsx: JSX.Element;
+}
+
+const Carousel: FC<CarouselProps> = (props) => {
+  const { items } = props;
+  const [activeSlide, setActiveState] = useState<number>(1);
+
+  const carouselItems = items.map<CarouselItem>((item, index) => ({
```
Comment (VolodymyrDevico) [inline src/components/Carousel/Carousel.tsx]: useMemo

File: src/App.tsx [hunk 3]
```diff
@@ -1,9 +1,15 @@
-import React from 'react';
+import Carousel from "./components/Carousel/Carousel";
+import UpcomingEventCard from "./components/UpcomingEventCard/UpcomingEventCard";
+import { FakeUpcomingEvents } from "./FakeUpcomingEvents";
```
Comment (turchin-aa) [inline src/App.tsx:3]: Volodymyr advised to avoid using semicolons, you can use VS code properties to fix it.

**Review by turchin-aa — COMMENTED** [review]
Good job boys! Please remove semicolons.

**Pull request merged**